microovn/client/client.go

```diff
@@ -157,11 +157,11 @@ func DisableService(ctx context.Context, c *client.Client, serviceName string) (
 
 // EnableService sends request to disable service with name as as specified in
 // "serviceName" argument.
-func EnableService(ctx context.Context, c *client.Client, serviceName string) (types.WarningSet, types.RegenerateEnvResponse, error) {
+func EnableService(ctx context.Context, c *client.Client, serviceName string, extraConfig *types.ExtraServiceConfig) (types.WarningSet, types.RegenerateEnvResponse, error) {
```

Comment:
I struggled a bit to draw a line where it would make sense to separate the commit. Although I tried to make the `--config` feature as reusable possible, some of the parts (e.g. [parsing the args into struct](https://github.com/mkalcok/microovn/blob/837db1e1b26fa124b2670a90d584d257d6edb17c/microovn/cmd/microovn/service_control.go#L145-L154)) would look weird or require placeholders if there's no subcommand that's actually using it.

However I can give it another go.

Reply:
This remains unsolved. Let me know if you think it's a blocker, because atm I don't see a nice way to split this commit.